dydx/client.py

```diff
@@ -386,6 +390,8 @@ def create_order(
             'salt': random.randint(0, 2**256)
         }
         order['typedSignature'] = utils.sign_order(order, self.private_key)
+        if triggerPrice:
```

Comment:
You can put this directly into the order object above. remove_nones on line 396 will remove it if None